torch/onnx/utils.py

```diff
@@ -113,6 +113,7 @@ def _split_tensor_list_constants(g, block):
 
 def _optimize_graph(graph, operator_export_type, _disable_torch_constant_prop=False, fixed_batch_size=False, params_dict=None):
     # Inline everyting
+    torch._C._jit_pass_onnx_preinline(graph)
```

Comment:
why does this pass need to be called twice? (once here and once in `_model_to_graph`)

Reply:
Initially I thought we can have it here, but it needs to be called earlier for jit.script modules. So I'll remove it from _optimize_graph